### ISSUE TITLE
Severe errors for baseboard electric heating in HVAC:Template:Zone:VAV:FanPowered

### ISSUE THREAD

**Issue by msfer86** [NREL/EnergyPlus#7377]
Issue overview
--------------
Using the HVAC:Template:Zone:VAV:FanPowered object with 'Baseboard Heating Type' = 'Electric', in the new EnergyPlus version 9.1.0, results in severe errors. Other baseboard types ('None' and 'HotWater') run correctly.
In the expanded file, in the respective ZoneHVAC:EquipmentList, it seems that the electric baseboard type (ZoneHVAC:Baseboard:Convective:Electric) and name are not recognized for the equipment 2 'Zone Equipment Object Type' and for the equipment 2 'Zone Equipment Name', respectively. Although these fields are correctly displayed in a text editor, they are skipped when opening the file with the EP-Launch.

Can this be somehow related to the new 'Zone Equipment Sequential Cooling Fraction' and 'Zone Equipment Sequential Heating Fraction' fields in the ZoneHVAC:EquipmentList object in v9.1.0? The problem did not occur in the previous EnergyPlus version (9.0.1). 

The error occurred in a user-defined file and also in the HVAC Template VAV Fan Powered example file: HVACTemplate-5ZoneVAVFanPowered.idf.

### Details
Some additional details for this issue (if relevant):
 - Windows 10 x64
 - Version of EnergyPlus: 9.1.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (list location of defect file here)  
[HVACTemplate-5ZoneVAVFanPowered-AddBaseboard.idf.txt](https://github.com/NREL/EnergyPlus/files/3373349/HVACTemplate-5ZoneVAVFanPowered-AddBaseboard.idf.txt)

 - [x] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@msfer86 Yes, there is a bug for this combination of equipment. The new fraction fields are missing from the ZoneHVAC:EquipmentList object just before the electric baseboard.
```
ZoneHVAC:EquipmentList,
  SPACE1-1 Equipment,                                      !- Name
  SequentialLoad,                                          !- Load Distribution Scheme
  ZoneHVAC:AirDistributionUnit,                            !- Zone Equipment Object Type
  SPACE1-1 ATU,                                            !- Zone Equipment Name
  1,                                                       !- Zone Equipment Cooling Sequence
  1,                                                       !- Zone Equipment Heating or No-Load Sequence
  ZoneHVAC:Baseboard:Convective:Electric,                  !- Zone Equipment Object Type
  SPACE1-1 Baseboard Heat,                                 !- Zone Equipment Name
  2,                                                       !- Zone Equipment Cooling Sequence
  2,                                                       !- Zone Equipment Heating or No-Load Sequence
  1,                                                       !- Zone Equipment Sequential Cooling Fraction
  1;                                                       !- Zone Equipment Sequential Heating Fraction
```

The workaround is to add two commas before the baseboard object type:
```
ZoneHVAC:EquipmentList,
  SPACE1-1 Equipment,                                      !- Name
  SequentialLoad,                                          !- Load Distribution Scheme
  ZoneHVAC:AirDistributionUnit,                            !- Zone Equipment Object Type
  SPACE1-1 ATU,                                            !- Zone Equipment Name
  1,                                                       !- Zone Equipment Cooling Sequence
  1,                                                       !- Zone Equipment Heating or No-Load Sequence
,
,
  ZoneHVAC:Baseboard:Convective:Electric,                  !- Zone Equipment Object Type
  SPACE1-1 Baseboard Heat,                                 !- Zone Equipment Name
  2,                                                       !- Zone Equipment Cooling Sequence
  2,                                                       !- Zone Equipment Heating or No-Load Sequence
  1,                                                       !- Zone Equipment Sequential Cooling Fraction
  1;                                                       !- Zone Equipment Sequential Heating Fraction
```

**Comment by mjwitte** [comment]
Turns out this problem existed in other terminal unit types. Full set of test files attached here along with group err file before and after fixes.
[7377-HVACTemplateBaseboard.zip](https://github.com/NREL/EnergyPlus/files/3632153/7377-HVACTemplateBaseboard.zip)

**Comment by mjwitte** [comment]
Closed via #7486
